modules/users/client/services/password-validator.client.service.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+
+// PasswordValidator service used for testing the password strength
+angular.module('users').factory('PasswordValidator', ['$window',
+  function ($window) {
+    var owaspPasswordStrengthTest = $window.owaspPasswordStrengthTest;
+
+    return {
+      getResult: function (password) {
+        var result = owaspPasswordStrengthTest.test(password);
+        return result;
+      },
+      getPopoverMsg: function () {
+        var popoverMsg = "Please enter a passphrase or password with greater than 10 characters, numbers, lowercase, upppercase, and special characters.";
```

Comment:
A good question. I just copied the directive to my meanjs-based project and got this. Is /client/services/ a new folder?

BTW Thanks for writing these!



> On 10 Sep 2015, at 14:21, Jason Loveland <notifications@github.com> wrote:
> 
> In modules/users/client/services/password-validator.client.service.js:
> 
> > @@ -0,0 +1,19 @@
> > +'use strict';
> > +
> > +// PasswordValidator service used for testing the password strength
> > +angular.module('users').factory('PasswordValidator', ['$window',
> > +  function ($window) {
> > +    var owaspPasswordStrengthTest = $window.owaspPasswordStrengthTest;
> > +
> > +    return {
> > +      getResult: function (password) {
> > +        var result = owaspPasswordStrengthTest.test(password);
> > +        return result;
> > +      },
> > +      getPopoverMsg: function () {
> > +        var popoverMsg = "Please enter a passphrase or password with greater than 10 characters, numbers, lowercase, upppercase, and special characters.";
> 
> Thanks @simison , I didn't see this show after running grunt jshint on that branch. Output from the command:
> 
> grunt jshint
> Running "jshint:all" (jshint) task
> >> 105 files lint free.
> 
> Done, without errors.
> 
> Any thoughts on why I don't get the errors?
> I can submit another PR to correct this though...
> 
> —
> Reply to this email directly or view it on GitHub.
>